libr/debug/p/debug_native.c

```diff
@@ -220,7 +220,9 @@ static int r_debug_native_continue_syscall (RDebug *dbg, int pid, int num) {
 	return ptrace (PTRACE_SYSCALL, pid, 0, 0);
 #elif __BSD__
 	ut64 pc = r_debug_reg_get (dbg, "PC");
-	return ptrace (PTRACE_SYSCALL, pid, (void*)(size_t)pc, 0);
+	errno = 0;
+	int retVal = ptrace (PTRACE_SYSCALL, pid, (void*)(size_t)pc, 0);
+	return 0==retVal;
```

Comment:
use `return retval == 0`. (first var and then value), also use shorter and non-camelcased variable names